### PR TITLE
Multiple improvements

### PR DESCRIPTION
**Pull request by masnick**
Major changes in this PR:

* Makes `fullUrl`, `id` and `reference` follow the guidance we received from FHIR-I to use `reference:#` syntax interpreted as an absolute URI within a Bundle, and apply consistently throughout the IG
* Fix `Immunization.status` to `completed`, and remove `statusReason` entirely
* Clean up the DM profiles for vaccinations and lab results to match the MS elements in the corresponding AD profiles

A number of other minor improvements are included as well.

### CODE REVIEW

File: input/fsh/invariants.fsh [hunk 1]
```diff
@@ -64,4 +64,13 @@ Severity:    #warning
...
 Expression:  "$this.length() < 20"
-Severity:    #warning
\ No newline at end of file
+Severity:    #warning
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+Invariant:   should-be-resource-uri
+Description: "IDs should use resource:# format"
+Expression:  "$this.matches('^resource:[0-9]+$')"
```
Comment (jmandel) [inline input/fsh/invariants.fsh:73]: Oh sweet!

File: input/fsh/profiles_bundle.fsh [hunk 2]
```diff
@@ -16,6 +19,14 @@ Description: "The bundle of resources that represents the clinical content of a
 * entry ^slicing.description = "Slicing based on the profile conformance of the entry"
 * entry and entry.resource MS
 
+* entry.fullUrl 1..1
+* entry.fullUrl MS
+* entry.fullUrl ^short = "Locally unique identifier like resource:0"
+* entry.fullUrl ^definition = "Identifier for the contained resource that is locally unique within this Bundle. The identifier SHOULD use `resource:#` format, where `#` is an integer incremented from 0 to _n_ within the Bundle."
+* entry.fullUrl ^comment = "n/a"
```
Comment (jmandel) [inline input/fsh/profiles_bundle.fsh:26]: Can `should-be-resource-uri` be applied here?

**Review by jmandel — APPROVED** [review]

File: input/pagecontent/index.md [hunk 3]
```diff
@@ -1,22 +1,24 @@
...
+**The goal of this IG is to constrain resources for use specifically in [SMART Health Cards].** This applies to the contents of both digital and paper Health Cards, including Health Cards produced via a Health Card-specific FHIR endpoint like `[base]/Patient/:id/$health-cards-issue`. This IG is not applicable to general purpose FHIR endpoints like `[base]/Patient/:id/Immunization`; these are governed by other IGs like [US Core](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-immunization.html).
+
+Note that this IG is not directly related to the [SMART App Launch Framework](http://www.hl7.org/fhir/smart-app-launch/). The name comes from [SMART Health IT](https://smarthealthit.org/), who also developed the [SMART Health Card] framework that this IG supports. SMART App Launch and SMART Health Cards are designed to work well together (the former being one of multiple methods for issuing the latter), but they don't have a hard dependency with each other.
```
Comment (jmandel) [inline input/pagecontent/index.md:21]:
```suggestion
Note that this IG is not directly related to the [SMART App Launch Framework](http://www.hl7.org/fhir/smart-app-launch/). Rather, this IG defines data profiles for use with the SMART Health Cards Framework. Both frameworks (SMART App Launch and SMART Health Cards) were initially developed by [SMART Health IT](https://smarthealthit.org/) at Boston Children's Hospital; these frameworks can be used together or independently.
```

File: input/pagecontent/index.md [hunk 4]
```diff
@@ -1,22 +1,24 @@
...
 2. Describes a minimal set of patient information (name and contact information) that is also included in the [SMART Health Card].
 
-The goal of this IG is to constrain resources for use specifically in [SMART Health Cards]. This applies to the contents of both digital and paper Health Cards, including Health Cards produced via a Health Card-specific FHIR endpoint like `[base]/Patient/:id/$health-cards-issue`. This IG is not applicable to general purpose FHIR endpoints like `[base]/Patient/:id/Immunization`; these are governed by other IGs like [US Core](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-immunization.html).
+**The goal of this IG is to constrain resources for use specifically in [SMART Health Cards].** This applies to the contents of both digital and paper Health Cards, including Health Cards produced via a Health Card-specific FHIR endpoint like `[base]/Patient/:id/$health-cards-issue`. This IG is not applicable to general purpose FHIR endpoints like `[base]/Patient/:id/Immunization`; these are governed by other IGs like [US Core](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-immunization.html).
```
Comment (p9g) [inline input/pagecontent/index.md:19]: "via Health Card-specific FHIR endpoint" is a little vague. The OperationDefinition for $health-cards-issue is not in this IG.  suggest "[Health Card-specific FHIR endpoint](https://smarthealth.cards/artifacts/operation-patient-i-health-cards-issue.json)"

File: examples/Scenario2Bundle.json [hunk 5]
```diff
@@ -1,12 +1,11 @@
 {
   "resourceType": "Bundle",
-  "id": "Scenario2Bundle",
+  "id": "Scenario1Bundle",
```
Comment (yunwwang) [inline examples/Scenario2Bundle.json:3]: Filename suggests this is Scenario2Bundle

**Pull request merged**